src/documentation/pages/components/pagination/examples/_additional-sizes_lumen_centurylink.vue

```diff
@@ -0,0 +1,243 @@
+<template lang="pug">
+  <ComponentExample title="Additional Sizes" id="additional_sizes_lumen_centurylink" :tabs="exampleTabs" padding="-p--0">
```

Comment:
To maintain consistency, you can update to kebab-case for id's. I see we are using kebab-case through out the code.